pkg/watcher/internal/metricsprovider/signalfx.go

```diff
@@ -206,3 +279,110 @@ func decodeMetricsPayload(payload interface{}) (float64, error) {
 	}
 	return timestampUtilisation[1].(float64), nil
 }
+
+/**
+Sample metricData payload:
+{
+  "data": {
+    "Ehql_bxBgAc": [
+      [
+        1600213380000,
+        84.64246793530153
+      ]
+    ],
+    "EuXgJm7BkAA": [
+	  [
+		1614634260000,
+		5.450946379084264
+     ]
+    ],
+    ....
+    ....
+  },
+  "errors": []
+}
+
+https://dev.splunk.com/observability/reference/api/metrics_metadata/latest#endpoint-retrieve-metric-timeseries-metadata
+Sample metaData payload:
+{
+    "count": 5,
+    "partialCount": false,
+    "results": [
+        {
+            "active": true,
+            "created": 1614534848000,
+            "creator": null,
+            "dimensions": {
+                "host": "gpma392019.gpf-prod.us-central1.gcp.dev.paypalinc.com",
```

Comment:
We should probably scrub this 😄

Reply:
Deleting this comment for obvious reasons, thanks for catching.